src/style.css

```diff
@@ -1,3 +1,76 @@
+/* #region Reset */
+
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
```

Comment:
it's better to set font-weight only for your text elements, for example links `<a>`